src/urbit/sur/chess.hoon

```diff
@@ -247,6 +248,8 @@
       [%result game-id=@dau result=chess-result]
       [%draw-offer game-id=@dau]
       [%draw-declined game-id=@dau]
+      [%draw-claim-available game-id=@dau]
+      [%special-draw-declined game-id=@dau]
```

Comment:
Want to check what I'm thinking before I start writing it:

1. I assume that the true/false value of `special-draw-available` should be sent to the frontend with each position update. 
2. To claim the draw while making a move, a player still needs to have their preference set to auto-claim. To claim it when receiving a threefold repetition (or >50th move) a player could auto-claim or click that button and send the `%claim-special-draw` poke. 
3. That means that `%claim-special-draw` needs to check whose turn it is because it can only be claimed on a player's own turn. A player should not be able to make a repeated move without auto-claiming the draw, then subsequently claim the draw before their opponent makes a move.

Reply:
Yes, that's all correct